### PR TITLE
Test that token is generated and has correct form (length and characters)

### PR DESCRIPTION
**Pull request by MilanPospisil**
In Collections -> API Token, token is generated after click on Load Token. We check with regexp if the token meets requirements (thus it is not something else or some error).

### CODE REVIEW

File: .directory [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Dolphin]
```
Comment (himdel) [inline .directory:1]: You'll probably want to add `.directory` to you global gitignore (https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)

File: test/cypress/integration/api_token.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    cy.login(adminUsername, adminPassword);
+  });
+
+  it('token is generated', () => {
+    cy.menuGo('Collections > API Token');
+    cy.contains('Load token').click();
+
+    cy.get('[aria-label="Copyable input"]')
+      .invoke('val')
+      .should('match', /[0-9a-f]{40}?/);
```
Comment (himdel) [inline test/cypress/integration/api_token.js:15]: the trailing question mark looks like a typo?

(LGTM apart from those 2 issues)

Comment (MilanPospisil) [reply test/cypress/integration/api_token.js:15]: Hmm...documentation says that after length there should be $, not ?. Probably mistake.

**Review by himdel — COMMENTED** [review]
LGTM, except there's still the `.directory` file

**Review by himdel — APPROVED** [review]
LGTM :+1: 

Time to remove `[WIP]`? :)

**Pull request merged**